lib/alphametics_cases.rb

```diff
@@ -0,0 +1,20 @@
+class AlphameticsCase < OpenStruct
+  def test_name
+    "test_#{description.gsub(' ', '_')}"
+  end
+
+  def do
```

Comment:
Any way to stop using 'do' as a method name?  I need to change Hamming, I think this is where this is coming from.